### DISCUSSION TITLE
表单设计器自定义组件拖拽后UI不更新

### DISCUSSION THREAD

**Discussion by huaxiangsirenbutong** [alibaba/formily#2835] [Q&A 2.x]
表单设计器自定义组件拖拽后UI不更新。。。。
如下图对自定义的上下结构对象组件进行拖拽后，UI只展示部分效果
![image](https://user-images.githubusercontent.com/37055241/154414707-8914fb33-4dc6-4d2c-b561-4d852a5349a3.png)

在点击JSONSCHEMA后或者保存后重新进入查看UI显示正常 操作如下图
![image](https://user-images.githubusercontent.com/37055241/154414623-9260dc80-feee-44d8-9483-b64704d66df1.png)
![image](https://user-images.githubusercontent.com/37055241/154414779-f16a78c5-f098-4599-84d0-37e69f4b37c8.png)
组件jsonSchema代码如下：
MobileSystemSingleObjectUpDownPage.Resource = createResource({
  icon: 'CustomMobileSearchBarIcon',
  title: {'zh-CN': '上下结构单对象', 'en-US': 'SingleObjectPage'},
  elements: [
    {
      componentName: 'Field',
      props: {
        "type": "void",
        "x-component": "FormLayout",
        "x-component-props": {},
        "name":"MobileSystemSingleObjectUpDownPage",
        "x-designable-id": "c0rqreb719p",
        "properties": {
          "qh9a3subexb": {
            "type": "object",
            "x-validator": [],
            "x-designable-id": "qh9a3subexb",
            // "x-index": 0
          },
          "myq36v9mj6z": {
            "type": "void",
            "x-component": "FormTab",
            "x-component-props": {},
            "x-designable-id": "myq36v9mj6z",
            "properties": {
              "k0dwwxtvv9t": {
                "type": "void",
                "x-component": "FormTab.TabPane",
                "x-component-props": {
                  "tab": "测试001"
                },
                "x-designable-id": "k0dwwxtvv9t",
                "properties": {
                  "gmwnjn5kew1": {
                    "type": "array",
                    "x-decorator": "FormItem",
                    "x-component": "MobileActionTable",
                    "x-validator": [],
                    "x-component-props": {},
                    "x-decorator-props": {},
                    "x-designable-id": "gmwnjn5kew1",
                  
                  }
                },
              
              }
            },
         
          }
        },
    
      },
    }

  ],
})
  求解决方案

**Comment by huaxiangsirenbutong** [comment]
@cncolder @s0ngyee

**Comment by huaxiangsirenbutong** [comment]
@janryWang

**Comment by janryWang** [comment]
把Resource的x-designable-id都删掉

**Comment by ironchengg** [comment]
我想问一下 这个问题解决了吗？我也是从git上把代码下到自己的项目上面，也发现在设计视图里面，网格布局的列不能拖拽，并且设置了跨列也不能直接显示出跨列的效果
